src/Common/Database/QueryUtils.php

```diff
@@ -83,6 +85,8 @@ public static function fetchTableColumn($sqlStatement, $column, $binds = array()
     public static function fetchSingleValue($sqlStatement, $column, $binds = array())
     {
         $records = self::fetchTableColumn($sqlStatement, $column, $binds);
+        // note if $records[0] is actually the value 0 then the value returned is null...
+        // do we want that behavior?
```

Comment:
probably not :)

Reply:
This is used in 42 other places.  I'm going to open a new issue for it as we'd need to test everywhere its used that it doesn't break functionality where someone may have worked around the behavior and returning a 0 will break their code... Shouldn't happen but better to be safe since its a core function.